charts/safe-client-gateway/values.yaml

```diff
@@ -108,6 +108,7 @@ service:
 ## @section Config Service Parameters
 
 config:
+  application_port: 80
```

Comment:
It's recommended to add documentation to values.yaml entries
In the future we will add a pipeline (already present in the `hoprd-operator` repository) which generates automatically the documentation for the helm-charts

Reply:
ok